package.json

```diff
@@ -170,6 +171,9 @@
     "winston": "^3.3.3",
     "ws": "^7.5.6"
   },
+  "optionalDependencies": {
+    "kafka-avro": "^3.1.1"
```

Comment:
That's a valid point. The slowness is significant.
Apart from that, as I mentioned in the PR description, since the installation of kafka-avro is optional, even where the installation is not possible, npm goes forward with the installation as far as it works, which causes an unnecessary amount of time spent for nothing.

Reply:
TBH, I'm 50/50 on including the kafka-avro tests. We (well, Willian) have invested time to build it. kafka-avro also was the request that lead to the instrumentation of node-rdkafka. OTOH, a "significant slowdown" (as Willian put) it plus the unfixable audit issue are clear downsides. 

Either way, it is good to have tested kavka-avro with this a couple of times on CI, even if we do not merge this particular integration test into main in the end.

Reply:
How about we install kafka-avro manually in our test only for v14?
Similar to what we do in our [ES tests](https://github.com/instana/nodejs/blob/main/packages/collector/test/tracing/database/elasticsearch_modern/test.js#L606)

This would I guess resolve both problems.

Reply:
I can take over this too :)

Reply:
> How about we install kafka-avro manually in our test only for v14?

I think this is a great idea! 👍